index.js

```diff
@@ -1,24 +1,140 @@
+    const { author } = this;
+
+    const newBook = {
+      title,
+      author,
+    };
+
+    this.books.push(newBook);
+  }
+
+  deleteBook() {
+    const item = this.button.parentNode;
+    const parent = item.parentNode;
+    parent.parentNode.removeChild(parent);
+
+    const { id } = this.button;
+
+    for (let i = 0; i < this.books.length; i += 1) {
+      if (id.includes(i)) {
+        this.books.splice(i, 1);
+      }
+    }
+  }
+}
+
+function storeForm(form) {
+  localStorage.setItem('form', JSON.stringify(form));
+}
 
 function displayBooks(book, index) {
   const content = `
-    <div>
-        <p>${book.title}</p>
-        <p>${book.author}</p>    
-        <button id ="delete${index}">Remove</button>
-        <hr>
-    </div>
+    <tr class="tableRow">
+        <td>"${book.title}" by ${book.author} </td>   
+        <td><button id ="delete${index}">Remove</button></td>
+    </tr>
     `;
   return content;
 }
 
-function storeForm(form) {s
-  localStorage.setItem('form', JSON.stringify(form));
+const addButton = document.getElementById('addBook');
+const newTitle = document.getElementById('bookTitle');
+const newAuthor = document.getElementById('authorName');
+const container = document.querySelector('.bodyh1');
+const navBar = document.querySelector('.nav-bar');
+
+for (let i = 0; i < books.length; i += 1) {
+  const book = displayBooks(books[i], i);
+  container.innerHTML += book;
 }
-s
\ No newline at end of file
+
+addButton.addEventListener('click', (e) => {
+  e.preventDefault();
+  if (newAuthor.value !== '' && newTitle.value !== '') {
+    const library = new Books(newAuthor.value, newTitle.value, books);
+    library.addBook();
+    storeForm(books);
+    window.location.reload();
+  }
+  document.getElementById('addNewForm').reset();
+});
+
+const deleteBtn = document.querySelectorAll('[id^="delete"]');
+deleteBtn.forEach((button) => {
+  button.addEventListener('click', () => {
+    const libraryOne = new Books(newAuthor.value, newTitle.value, books, button);
+    libraryOne.deleteBook();
+    storeForm(books);
+  });
+});
+
+// SPA
+
+const navHome = document.querySelector('.nav-home');
+const navBooks = document.querySelector('.nav-books');
+const navContact = document.querySelector('.nav-contact');
+
+const homeSection = document.querySelector('.home');
+const bookSection = document.querySelector('.books');
+const contactSection = document.querySelector('.contacts');
+
+navHome.addEventListener('click', () => {
+  navHome.style.color = 'blue';
+  navBooks.style.color = 'black';
+  navContact.style.color = 'black';
+  homeSection.style.display = 'flex';
+  bookSection.style.display = 'none';
+  contactSection.style.display = 'none';
+});
+navBooks.addEventListener('click', () => {
+  navHome.style.color = 'black';
+  navBooks.style.color = 'blue';
+  navContact.style.color = 'black';
+  homeSection.style.display = 'none';
+  bookSection.style.display = 'flex';
+  contactSection.style.display = 'none';
+});
```

Comment:
- [ ] Just as the previous reviewer highlighted, when the contact link is clicked, the contact section is expected to show on the page but it's not at the moment and I found out that you forgot to add this code that shows the contact section just as you did for the other two sections:
- [ ] Add these codes below line 108 so things will work fine as expected 👍🏽

```js
navContact.addEventListener("click", () => {
  navHome.style.color = "black";
  navBooks.style.color = "blue";
  navContact.style.color = "black";
  homeSection.style.display = "none";
  bookSection.style.display = "none";
  contactSection.style.display = "block";
});
``

Reply:
Thank you for the kind words. Please check it again 👏
<img src="https://media0.giphy.com/media/2fD2lWKJVNRTCmV7GR/giphy.gif"/>

 You can find this project here:  GitHub: [@kayjeee](https://github.com/kayjeee/bookstore).  Please leave a star it helps. Thanks for checking out my code!